src/main/java/no/unit/nva/cristin/projects/model/cristin/Unit.java

```diff
@@ -0,0 +1,16 @@
+package no.unit.nva.cristin.projects.model.cristin;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import java.util.Map;
+
+@SuppressWarnings("PMD.ShortClassName")
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public class Unit {
+
+    public String cristinUnitId;
```

Comment:
Could we make a PR for fixing the issue with the public fields?

Reply:
And then fix the public fields after this current PR is merged?

Reply:
yes